stor/dx.py

```diff
@@ -1087,6 +1087,11 @@ def stat(self):
         return dxpy.DXFile(dxid=self.canonical_resource,
                            project=self.canonical_project).describe()
 
+    @property
+    def content_type(self):
+        """Get content type for DXObject. Returns empty string if not present or is project/"""
+        return self.stat().get('media') or ''
```

Comment:
Note that `DXPath.stat()` raises error for a folder path. Do we also want that behavior for `content_type`?

Reply:
yeah that seems reasonable